src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -840,17 +887,22 @@ impl<T: AstInfo> AstDisplay for CreateSourceConnection<T> {
                     f.write_str(")");
                 }
             }
-            CreateSourceConnection::Kinesis { arn } => {
-                f.write_str("KINESIS ARN '");
+            CreateSourceConnection::Kinesis { connection, arn } => {
+                f.write_str("KINESIS CONNECTION ");
```

Comment:
the `KAFKA` one above doesn't have the CONNECTION word? it might not matter

Reply:
Great catch; ty